src/fairchem/core/trainers/ocp_trainer.py

```diff
@@ -233,10 +237,21 @@ def train(self, disable_eval_tqdm: bool = False) -> None:
         if self.config.get("test_dataset", False):
             self.test_dataset.close_db()
 
+    def _raw_output(self, target_key: str, prediction: torch.Tensor, batch: Batch):
```

Comment:
nit - we can add these lines directly into `_forward`. This method name is rather confusing.